go.mod

```diff
@@ -96,6 +96,7 @@ require (
 	k8s.io/api v0.18.8
 	k8s.io/apimachinery v0.18.8
 	k8s.io/client-go v0.18.8
+	k8s.io/klog v1.0.0
```

Comment:
We want to use k8s.io/klog/v2 through the codebase, so to fix this I would:

1. Replace any imports of `k8s.io/klog` with `k8s.io/klog/v2`
2. Run `go mod tidy` to remove this dependency from go.mod